proposals/tsc-voting-procedures.md

```diff
@@ -0,0 +1,199 @@
+
+## Overview
+
+This proposal establishes standardized voting procedures for the Hiero Technical Steering Committee (TSC) to improve decision-making efficiency while maintaining transparency and adherence to charter requirements.
+
+## Voting Methods
+
+### 1. In-Meeting Votes
+- **When Used**: For decisions during TSC meetings with quorum present
+- **Threshold**: Simple majority of members present
+- **Recording**: Documented in meeting minutes with vote counts
+
+### 2. Asynchronous Votes via GitHub Pull Request
+- **When Used**: 
+  - When quorum cannot be achieved in meetings
+  - For decisions requiring extended review/discussion
+  - For charter-mandated votes (e.g., license exceptions)
+- **Location**: Pull requests in the `proposals/` directory
+- **Notification**: All TSC members must be notified via:
+  - GitHub PR mention (@username)
+  - Slack notification in TSC channel
+
+## Voting Process
+
+### Step 1: Discussion (Optional)
+- Create a GitHub Issue for complex proposals requiring discussion
+- Label with "tsc-discussion"
+- No formal timeline - continue until consensus on proposal language
+
+### Step 2: Proposal Creation
+1. Create a new file in `proposals/` directory following the template
+2. File naming convention: `YYYY-MM-DD-brief-description.md`
+3. Open a Pull Request with the finalized proposal
+4. **Proposal is immutable once PR is opened** - no changes allowed
+5. Link to discussion issue if one exists
+
+### Step 3: Voting Period
+- **Duration**: 7 calendar days maximum
+- **Purpose**: Voting only (discussion happens in issue, not PR)
+- **Voting Method**: 
+  - Approve PR = Yes vote
+  - Request Changes = No vote
+  - No response = Abstention
+- **Comments**: Brief rationale only - substantive discussion in linked issue
+
+### Step 4: Resolution
+- **Passing Threshold**: 
+  - Standard decisions: >50% of eligible voting members
+  - Charter-specified decisions: As defined in charter (e.g., 2/3 for license exceptions)
+- **Early Resolution**: Vote concludes immediately when:
+  - More than 50% of eligible members vote "Yes" (proposal passes)
+  - More than 50% of eligible members vote "No" (proposal fails)
+  - Mathematically impossible to change outcome
+- **Quorum**: Minimum participation of 50% of eligible voting members
+- **Recording**: Merge PR if passed, close if failed
+
+## Voting Eligibility
+
+### Prerequisites
+- All TSC members must have a GitHub account to participate in voting
+
+### Active Voting Members
+- All TSC members listed in README.md
+- Have not missed 3 consecutive TSC meetings
+- Have GitHub account with appropriate repository access
+
+### Eligibility Tracking
+- Maintain eligibility status in `tsc-voting-eligibility.md`
+- Update after each meeting
+- Include:
+  - Member name
+  - GitHub username
+  - Voting status (active/suspended)
+  - Last meeting attended
+
+## Special Voting Scenarios
+
+### Urgent Decisions
+- **Definition**: Time-sensitive matters requiring faster resolution
```

Comment:
Who defines that a task is URGENT? I would like to remove this section for now and discuss it once we have a general working process